lib/Sema/TypeCheckAttr.cpp

```diff
@@ -795,6 +795,14 @@ void AttributeChecker::visitAccessControlAttr(AccessControlAttr *attr) {
           .fixItRemove(attr->getRange());
       }
     }
+
+    if (auto FD = dyn_cast<FuncDecl>(D)) {
+      if (!FD->isObjC() && attr->getAccess() == AccessLevel::Open) {
+        diagnose(attr->getLocation(), diag::access_control_non_objc_open_func)
+            .fixItReplace(attr->getRange(), "public ");
```

Comment:
Either there is an extra space here or there is a missing space in the `"public"` below this. But it seems like they should be consistent.

Reply:
Also, related to my comment in the other PR; I think this should apply to properties too and not just methods.

Reply:
Thanks! I removed the extra space and applied this to properties as well :)